docs/release-notes/8.3.asciidoc

```diff
@@ -0,0 +1,82 @@
+* Creates landing pages for Dashboards and Threat hunting, for now only accessible via URL ({pull}130905[#130905]).
+* Adds a “User risk” tab to the User details flyout ({pull}130256[#130256]).
+* Adds an “Authentications” tab to the User details flyout ({pull}129456[#129456]).
+* Adds the ability to investigate Osquery results in Timeline ({pull}128596[#128596]).
+* Allows multiple alerts to be added to a case ({pull}130958[#130958]).
+* Adds the option to delete case comments multiple alerts to be added to a case ({pull}130254[#130254]).
+* Provides an option to select a severity level for a case ({pull}131626[#131626]).
+* Adds the *Alerts* tab to cases ({pull}131883[#131883]).
+* Adds the *Average time to close* metric to the Cases page ({pull}131909[#131909]).
+
+[discrete]
+[[bug-fixes-8.3.0]]
+==== Bug fixes and enhancements
+* Separates array values in the Alerts table with commas ({pull}133297[#133297]).
+* Exposes the EQL search configurations `event_category_field`, `tiebreaker_field`, and `timestamp_field` through the rules API and UI for Event Correlation rules ({pull}132247[#132247]).
+* Adds the *Session ID* field to the *Highlighted fields* section on the Alert details flyout ({pull}132219[#132219]).
+* Introduces a new design for the main navigation menu ({pull}132210[#132210], {pull}131437[#131437]).
+* Adds Dashboards and Threat Hunting Landing pages ({pull}130905[#130905]).
+* Introduces the Detections & Response dashboard ({pull}130670[#130670], {pull}129021[#129021], {pull}131029[#131029], {pull}128335[#128335], {pull}128087[#128087]).
+** Update Detections Response view with pagination and opening numbers in Timeline ({pull}131828[#131828], {pull}131029[#131029]).
+* Allows highlighted fields to be investigated in Timeline ({pull}131255[#131255]).
+* Adds support to Osquery for multi-line queries ({pull}131224[#131224]).
+* Enforces the `All` global privilege setting on Osquery ({pull}130523[#130523]).
+* Adds the *Run Osquery* option to the *More actions* menu (*...*) in the Alerts table ({pull}131790[#131790]).
+* Improves the performance of these actions on the bulk rule actions endpoint ({pull}130924[#130924]).
+** `add_tags`
+** `delete_tags`
+** `set_tags`
+** `add_index`
+** `delete_index`
+** `set_index_patterns`
+** `set_timeline_template`
+* Adds the *User risk* tab to the user details page ({pull}130256[#130256]).
+* Fixes a bug that caused the rule details page to crash when users opened a deleted or non-existent rule ({pull}133867[#133867]).
+* Allows threshold alerts to be investigated in Timeline if filters are not provided ({pull}133733[#133733]).
+* Prevents events from being added to cases from Timeline ({pull}133410[#133410]).
+* Fixes All Users and All Hosts page should reset the current page when sorting ({pull}133111[#133111]).
+* Removes the filter and investigate in Timeline options from the Elastic Agent status in Highlighted fields ({pull}132829[#132829], {pull}132586[#132586]).
```

Comment:
```suggestion
* Removes the Filter and Investigate in Timeline options from the Elastic Agent status in Highlighted fields ({pull}132829[#132829], {pull}132586[#132586]).
```
Should we bold "Filter" and "Investigate in Timeline" since they are clickables?

Reply:
These are the shorthand versions of the button names, figured they'd be easier to read and more succinct. The filter function in and out buttons aren't named in the UI and the add to timeline option is actually named **Add to timeline investigation**.